eurofaang_ri_be/eurofaang_ri_be/settings.py

```diff
@@ -113,6 +113,13 @@
     },
 ]
 
+PASSWORD_HASHERS = [
+    "django.contrib.auth.hashers.PBKDF2PasswordHasher",
+    "django.contrib.auth.hashers.PBKDF2SHA1PasswordHasher",
+    "django.contrib.auth.hashers.Argon2PasswordHasher",
+    "django.contrib.auth.hashers.BCryptSHA256PasswordHasher",
+    "django.contrib.auth.hashers.ScryptPasswordHasher",
+]
```

Comment:
I've just tested it and looks like it doesn't have any effect on the password hashing logic. Do we really need to have it here?

Reply:
we don't really need it as this algorithm is used by default

https://docs.djangoproject.com/en/5.0/topics/auth/passwords/

I thought it was make sense to specify it more explicitly in the settings